core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -294,8 +302,10 @@
    * Used for setting configuration in integration tests.
    */
   public static final class Name {
+    public static final String CONF_DIRECTORY = "alluxio.conf.directory";
     public static final String DEBUG = "alluxio.debug";
     public static final String HOME = "alluxio.home";
+    public static final String INSTALL_HOME = "alluxio.install.home";
```

Comment:
also @calvinjia @jsimsa , I remember we talked before whether we should use "folder" or "directory" in property name consistently in the future. for now I prefer "directory", any preference?